src/FirebirdSql.EntityFrameworkCore.Firebird/Scaffolding/Internal/FbDatabaseModelFactory.cs

```diff
@@ -343,11 +344,18 @@ private void GetIndexes(DbConnection connection, IReadOnlyList<DatabaseTable> ta
 							IsUnique = reader.GetBoolean(1),
 						};
 
-						foreach (var column in reader.GetString(2).Split(','))
+						foreach (var column in reader.GetString(3).Split(','))
 						{
 							index.Columns.Add(table.Columns.Single(y => y.Name == column.Trim()));
 						}
 
+						if (reader.GetBoolean(2))
+						{
+							var isDescending = new bool[index.Columns.Count];
+							isDescending[0] = true;
```

Comment:
Shouldn't this be set to `true` for all elements of the array?

Reply:
Correct, missed this while modifying the PR after your first review.